index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<header id="header">
```

Comment:
Kindly change the link from <header> to <head>. The head tag is the right tag to contain the link tag, the meta tag and the title tag.